contracts/upgradeable_contracts/arbitrary_message/MessageDelivery.sol

```diff
@@ -66,14 +66,16 @@ contract MessageDelivery is BasicAMB, MessageProcessor {
         // 79 = 4 + 20 + 8 + 20 + 20 + 4 + 1 + 1 + 1
         header = new bytes(79 + srcChainIdLength + dstChainIdLength);
 
+        uint256 datatype = _messageDatatype();
```

Comment:
Should we increase the version of AMB contracts now in order to reflect that their functionality was changed?

Reply:
Bumped AMB contracts version to `5.4.0`